website/integrations/services/fortigate-ssl/index.md

```diff
@@ -0,0 +1,239 @@
+>
+> -   Fortigate: 7.2.8
+> -   authentik: 2024.2.2
+
+## Assumptions
+
+-   You know how to configure an SSLVPN in a FortiGate
+-   You have already a certificate for signing and encryption uploaded to both authentik and the FortiGate.
+-   You already have a working SSLVPN (either portal or tunnel) and is just changing authentication from what you are using today to authentik SAML.
+
+The following placeholders will be used:
+
+-   `saml.sp.name` = The name that will be the SAML SP configuration in the FortiGate
+-   `fgt.cert` = Fortigate certificate for signig and encrypting
+-   `service.company` = This is the FQDN of the firewall, if your sslvpn portal is not on TCP port 443, then add the port like: fortigate.mydomain.tld:10233
+-   `authentik.company` = This is the FQDN of your authentik installation
+-   `app.slug.name` = The application slug that you decided upon
+-   `ak.cert` = The authentik remote certificate you have uploaded before starting the guide.
+-   `fgt.user.group` = This will be the name of the user group in your Fortigate that you will use in your SSLVPN portal mapping and Firewall rules
+-   `ak.user.group` = This is the user group name that you will use in authentik if you plan on limiting access to the sslvpn via groups.
+
+## FortiGate configuration
+
+### Preparation
+
+-   Decide on an application name (slug) e.g. fgtsslvpn that you will use in authentik later.
+
+### Setup SAML SP
+
+1.  ssh to the Fortigate (If you are using vdom change to the correct vdom)
+2.  copy the config below to your prefered editor and change the placeholders to your settings, then paste it into the Fortigate
+
+> [!NOTE]
+> Some are https and some are http, that is on purpose, and as described by FortiNet.
+
+```
+config user saml
+    edit "saml.sp.name"
+        set cert "fgt.cert"
+        set entity-id "http://service.company/remote/saml/metadata/"
+        set single-sign-on-url "https://service.company/remote/saml/login"
+        set single-logout-url "https://service.company/remote/saml/logout"
+        set idp-entity-id "https://authentik.company"
+        set idp-single-sign-on-url "https://authentik.company/application/saml/app.slug.name/sso/binding/redirect/"
+        set idp-single-logout-url "https://authentik.company/application/saml/app.slug.name/slo/binding/redirect/"
+        set idp-cert "ak.cert"
+        set user-name "http://schemas.goauthentik.io/2021/02/saml/username"
+        set group-name "http://schemas.xmlsoap.org/claims/Group"
+        set digest-method sha256
+    next
+end
+```
+
+### Add the saml single signon to a use group
+
+This will limit who can login via authentik saml. It will match on ak.user.group which is the group you will setup in authentik later, and only allow users of that group to login. In essence it provides the same functionality as returning a user-group via radius, and matching on the user group.
```

Comment:
```suggestion
This will limit who can login via authentik SAML. It will match on `ak.user.group` which is the group you will set up in authentik later, and only allow users of that group to login. In essence it provides the same functionality as returning a user-group via Radius, and matching on the user group.
```